README.md

```diff
@@ -20,6 +20,25 @@ http://github.com/richrd/suplemon
 
 ![Suplemon in action](http://www.suplemon.com/assets/suplemon-v0.1.51-multi-cursor-editing.gif)
 
+## Features
+ * Terminal text editing with style
+ * Proper multi cursor editing, Sublime Text style.
+ * Syntax highlighting
+ * Autocomplete (based on words in the files that are open)
+ * Easy Undo/Redo (Ctrl + Z, Ctrl + Y)
+ * Copy & Paste, with multi line support (and native clipboard support on X11 / Unix)
+ * Multiple files in tabs
+ * Powerful Go To feature for jumping to files and lines
+ * Find, Find next and Find all (Ctrl + F, Ctrl + D, Ctrl + A)
+ * Custom keyboard shortcuts (and easy-to-use defaults)
+ * Mouse support
+ * Restores cursor positions in when reopenning files.
+ * Extensions (easy to write your own)
+ * Lots more...
+
+
+## Caveats
+ * Currently no built in selections (regions). To copy part of a line select it with your mouse and use <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>C</kbd>
```

Comment:
Currently we don't have any specific issue about that. I just thought that I should probably point out the lack of regions and a workaround. I haven't yet planned to implement them in the near future so I probably won't create the issue either.